### DISCUSSION TITLE
Automatically resolve alerts that don't matter

### DISCUSSION THREAD

**Discussion by mcandre** [community/community#17226] [Code Security] [answered]
I got two different inactionable alerts from Dependabot this week for that stupid nokogiri dependency. This time, both alerts remarked that the vulnerability has already been fixed and does not require any action on my part as a consumer of nokogiri. Okay.... so go ahead and automatically dismiss the alert then. Don't leave useless alerts around, which distract from more critical, actionable issues.

**Comment by erinhav** [comment, marked as answer]
A fix for this has been rolled out, so this won't happen again for future alerts. However, we still need to figure out a backfill strategy so we can go and close open alerts which should have been closed. We're very sorry for the inconvenience in the meantime.
